lightly/api/api_workflow_upload_embeddings.py

```diff
@@ -29,29 +32,37 @@ def _get_csv_reader_from_read_url(self, read_url: str):
 
         return reader
 
+    def set_embedding_id(self):
```

Comment:
The naming is confusing here: It says "set" but I can't set it to anything and I also don't know to what it's being set

Reply:
What do we need this function for?

Reply:
We need it for sampling:
https://github.com/lightly-ai/lightly/blob/072fe3fd8d2244540bcc4bebef22de9fbcdc2101/lightly/api/api_workflow_sampling.py#L84-L93

Reply:
I have renamed the method to `set_latest_embedding_id` and added a docstring.

Reply:
Your renaming doesn't resolve the problem, it's still a setter without an argument which is really strange. Now it just seems like I can "set the latest id" but I want to set the local id to the latest id on the server. For example "set_embedding_id_to_latest"